security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolation.java

```diff
@@ -211,4 +213,16 @@ public double getValue() {
             return null;
         }
     }
+
+    public String toString() {
+        String base = "Subject id: " + this.subjectId + ", Subject name: " + this.subjectName + ", limitType: " +
+                this.limitType.toString() + ", limit: " + this.limit + ", limitName: " + this.limitName +
+                ", acceptableDuration: " + this.acceptableDuration + ", limitReduction: " + this.limitReduction +
+                ", value: " + this.value;
+        if (side != null) {
+            return base + ", side: " + this.side.toString();
+        } else {
+            return base + ", side: null";
+        }
```

Comment:
Great: you put an exemple of what I said just before!

Simply write something like this : ` + ", side: " + this.side`